apps/documentation/src/routes/documentation/tutorial/components.mdx

```diff
@@ -58,13 +57,12 @@ Now that the link is done, let’s import it into the `index.tsx` file
 ++  import PokemonLink from '../components/PokemonLink'
 
 // ...
-	<ul style={{ flexWrap: "wrap", display: "flex", gap: 10 }}>
---      {pokemons.map((pokemon) => {
---        return pokemon.name;
-++      {pokemons.map((pokemon, i) => {
-++        return <PokemonLink pokemon={pokemon} id={i + 1} key={i} />;
-      })}
-	</ul>
+	    <ul style={{ flexWrap: "wrap", display: "flex", gap: 10 }}>
+        {pokemons.map((pokemon, i) => (
```

Comment:
This has wrong indentation

Reply:
The indention I used is consistent with the file inside `examples/tuono-tutorial`:

<img width="681" alt="Screenshot 2025-01-28 at 18 37 39" src="https://github.com/user-attachments/assets/f9f5ea7f-aa80-42c4-a608-b4cda017fbd1" />

Reply:
Yeah but for some reason it breaks on the documentation website